doc/tutorials/calib3d/camera_multiview_calibration/multiview_calibration.markdown

```diff
@@ -0,0 +1,226 @@
+Multi-view Camera Calibration Tutorial
+==================
+
+The following sections describes the individual algorithmic steps of the overall multi-camera calibration pipeline:
+
+Briefly:
+----
+1. Calibrate intrinsics parameters (intrinsic matrix and distortion coefficients) for each camera independently.
+2. Calibrate pairwise cameras (using stereo calibration) via intrinsics parameters from step 1.
+3. Do global optimization using all cameras simultaneously to refine extrinsic parameters.
+
+Steps in detail:
+----
+1. If the intrinsics are not provided, the calibration procedure starts calibrating them independently for each camera.
+* a\. If input is a combination of fisheye and pinhole cameras, then fisheye images are calibrated with the default OpenCV calibrate function. The reason is that stereo calibration in OpenCV does not support a mix of fisheye and pinhole cameras. The following flags are used;
+* * i\. CALIB_RATIONAL_MODEL - it extends default (5 coefficients) distortion model and returns more parameters.
+* * ii\. CALIB_ZERO_TANGENT_DIST - it zeroes out tangential distortion coefficients, since the fisheye model does not have them.
+* * iii\. CALIB_FIX_K5, CALIB_FIX_K6 - it zeroes out the fifth and sixth parameter, so in total 4 parameters are returned.
```

Comment:
Please use references. It'll be even better to have reference on detailed camera model description with flags rather then individual flags without formulas.

Reply:
fixed